lighthouse-core/test/audits/__snapshots__/metrics-test.js.snap

```diff
@@ -2,44 +2,44 @@
 
 exports[`Performance: metrics evaluates valid input (with lcp) correctly 1`] = `
 Object {
-  "estimatedInputLatency": 1324,
+  "estimatedInputLatency": 350,
   "estimatedInputLatencyTs": undefined,
-  "firstCPUIdle": 14907,
-  "firstCPUIdleTs": 1671236823075,
-  "firstContentfulPaint": 4702,
-  "firstContentfulPaintTs": 1671226617803,
-  "firstMeaningfulPaint": 4702,
-  "firstMeaningfulPaintTs": 1671226617803,
-  "interactive": 14907,
-  "interactiveTs": 1671236823075,
-  "largestContentfulPaint": 15024,
-  "largestContentfulPaintTs": 1671236939268,
-  "maxPotentialFID": 1808,
-  "observedDomContentLoaded": 14907,
-  "observedDomContentLoadedTs": 1671236823075,
-  "observedFirstContentfulPaint": 4702,
-  "observedFirstContentfulPaintTs": 1671226617803,
-  "observedFirstMeaningfulPaint": 4702,
-  "observedFirstMeaningfulPaintTs": 1671226617803,
-  "observedFirstPaint": 4702,
-  "observedFirstPaintTs": 1671226617803,
-  "observedFirstVisualChange": 4685,
-  "observedFirstVisualChangeTs": 1671226600754,
-  "observedLargestContentfulPaint": 15024,
-  "observedLargestContentfulPaintTs": 1671236939268,
-  "observedLastVisualChange": 4685,
-  "observedLastVisualChangeTs": 1671226600754,
-  "observedLoad": 40464,
-  "observedLoadTs": 1671262379989,
+  "firstCPUIdle": 10075,
+  "firstCPUIdleTs": undefined,
+  "firstContentfulPaint": 8828,
+  "firstContentfulPaintTs": undefined,
```

Comment:
FCP, FMP, SI, and TTI flipped to undefined. i don't think that's expected is it?

Reply:
we went from devtools to simulate so i think these are expected to be undefined

Reply:
oic.